### ISSUE TITLE
arlequinWrite() with haploid microsatellite data

### ISSUE THREAD

**Issue by Gillous** [EricArcher/strataG#38]
Hi Eric, 

I was confronted to an issue relating to my haploid microsatellite data, which is not processed by the arlequinWrite() function, as the ploidy is set to >1 in the code. For haploid data, the ploidy is ==1 and then, the arlequinWrite() function is switching to the  "FREQUENCY" type.

My request would to use the "MICROSAT" type while ploidy ==1. 

Output format for Arlequin is the same as already written in your code for "MICROSAT". The easiest way would be to set up a switch in the command line; for instance: arlequinWrite(g, file = NULL, type= "microsat", ploidy = "haploid"). What do you think?

Warm regards, 

Gilles

**Comment by EricArcher** [comment]
Example in https://github.com/EricArcher/strataG/issues/37#issuecomment-667035061

**Comment by EricArcher** [comment]
Apologies for the delay in getting to this. I have finally found a little bit of time to start working on issues in the package.
I've added a `haploid.microsat` argument to `writeArlequin()` that will override the default of DataType=FREQUENCY if haploid data is in the types object. I tested it on your dataset with the following and it seemed to work properly.

```
library(poppr)
load("vn14GEOpop")
gt <- genind2gtypes(genclone2genind(vn14GEOpop))
arlequinWrite(gt, file = "vn14GPSpop.arp", haploid.microsat=T)
```
